pkg/workload/rand/rand.go

```diff
@@ -67,10 +85,11 @@ var randMeta = workload.Meta{
 		g.flags.StringVar(&g.tableName, `table`, ``, `Table to write to`)
 		g.flags.IntVar(&g.batchSize, `batch`, 1, `Number of rows to insert in a single SQL statement`)
 		g.flags.StringVar(&g.method, `method`, `upsert`, `Choice of DML name: insert, upsert, ioc-update (insert on conflict update), ioc-nothing (insert on conflict no nothing)`)
-		g.flags.Int64Var(&g.seed, `seed`, 1, `Key hash seed.`)
+		g.flags.Int64Var(&g.seed, `seed`, defaultRandomSeed(), `Key hash seed.`)
```

Comment:
Nice! I wasn't aware we were defaulting to `1`, so I checked other workloads. They seem to follow the same pattern, e.g., https://github.com/cockroachdb/cockroach/blob/master/pkg/workload/tpcc/tpcc.go#L187

I wonder why?